recipes-devtools/ruby/ruby_2.5.3.bbappend

```diff
@@ -0,0 +1 @@
+SRC_URI_remove = "file://CVE-2018-1000073.patch"
```

Comment:
Why is needed to remove this patch from ruby?

Reply:
Some recipes from meta-cloud-services are not pretty up to date and fails due to this fix. In particular packages from meta-cloud-services/meta-openstack.

Of course this is a temporary fix done time ago (note this PR is still WIP). Maybe this is not needed anymore.

Reply:
Not needed anymore. Removed.